core/src/identity.rs

```diff
@@ -32,12 +32,15 @@
 //! (e.g. [ed25519 binary format](https://datatracker.ietf.org/doc/html/rfc8032#section-5.1.5)).
 //! All key types have functions to enable conversion to/from their binary representations.
 
-#[cfg(feature = "ecdsa")]
+#[cfg(any(feature = "ecdsa", docsrs))]
```

Comment:
Why are we combining these with `any`?

Shouldn't the regular feature be enough?